tests/lac/bicgstab_large.cc

```diff
@@ -31,6 +31,7 @@ main()
 {
   initlog();
   deallog << std::setprecision(4);
```

Comment:
Would it make sense to drop this `std::setprecision` or to set it to say 10 digits so that numdiff can do something about roundoff errors?

Reply:
The problem is that we print a residual that ought to be zero in exact arithmetic, but turns out to be 1e-3 or something (above the absolute/relative threshold of numdiff), because we've scaled by matrix by `1e10`. I don't see a simple way to get this back to tolerance, apart from disabling the print of the residual to the logstream.